RELEASE.CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### Release 2024-06-21
+* **Amazon.Lambda.Templates (7.2.0)**
+  * Updated Amazon.Lambda.Annotations dependency to 1.5.0 in the Message Processing Framework Visual Studio blueprint. Added the `SQSEvent` attribute to set up SQS event source mapping for the message handler Lambda function.
```

Comment:
I'd drop "Visual Studio" since this is technically a `dotnet new` template as well.

Reply:
Done